src/sentry/auth/email.py

```diff
@@ -86,10 +86,11 @@ class HasOrgMembership(ResolutionStep):
         def apply(self, candidates: Collection[UserEmail]) -> Iterable[UserEmail]:
             if not self.parent.organization:
                 return ()
-            query = OrganizationMember.objects.filter(
-                organization=self.parent.organization, user_id__in=[ue.user_id for ue in candidates]
+            candidate_ids = [ue.user_id for ue in candidates]
+            member_summaries = organization_service.get_member_summaries_by_ids(
+                organization_id=self.parent.organization.id, user_ids=candidate_ids
             )
```

Comment:
It isn't safe to add an RPC method and call it in the same pull request. The regions deploy well before control does. You'll need to add the RPC method first, deploy those changes and then do a separate set of changes that introduce a caller to the new method.

Reply:
Makes sense, I'll update the PR

Reply:
moved caller to https://github.com/getsentry/sentry/pull/64105/files